### PR TITLE
Create optional sig-compute conformance lane

### PR DESCRIPTION
**Pull request by xpivarc**
**What this PR does / why we need it**:
This will provide feedback for the conformance tests. (runtime, flakiness and other attributes)

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by xpivarc** [comment]
@brianmcarey I linked the Kubevirt/Kubevirt change

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml [hunk 1]
```diff
@@ -1986,6 +1986,51 @@ presubmits:
...
+        - /usr/local/bin/runner.sh
+        - /bin/sh
+        - -c
+        - automation/test.sh
+        env:
+        - name: TARGET
+          value: k8s-1.31-sig-compute-conformance
+        - name: KUBEVIRT_PSA
+          value: "true"
+        - name: KUBEVIRT_WITH_ETC_IN_MEMORY
```
Comment (brianmcarey) [inline github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:2021]: Can you update to use the preset `preset-kubevirtci-etcd-in-mem: "true"` ?

**Review by brianmcarey — COMMENTED** [review]
Just one nit inline  - we merge this once that is addressed. 

The target is not available in kubevirt/kubevirt yet but we can use this job to test the PR that introduces it.

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml (hunk 1 above)
Comment (xpivarc) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml]: Done.

**Review by brianmcarey — APPROVED** [review]
/approve 

thanks @xpivarc

**Pull request merged**